### DISCUSSION TITLE
How can i load subsources using node-fetch?

### DISCUSSION THREAD

**Discussion by mai-maggie** [node-fetch/node-fetch#1645] [Q&A] [answered]
so I try to get the HTML from a URL using node-fetch but I can't get the video subsources. I check the options part in readme but didn't see any mentions about how to fetch the video sources.

**Comment by GabenGar** [comment, marked as answer]
`fetch` isn't responsible for parsing HTML, you need to use the likes of `jsdom` to emulate a DOM structure of the HTML document on the server.

**Reply by jimmywarting** [reply]
Yep, the best thing you can do is just using `text = await response.text()`
then i would perhaps just use some regular expression to get the video source instead of depending on some large jsdom lib

**Reply by GabenGar** [reply]
Yeah parsing html with regexp is a great idea.
